src/DevMenu/ui/Bladeburner.tsx

```diff
@@ -18,13 +18,11 @@ import { Player } from "@player";
 import { CityName } from "@enums";
 import { Skills as AllSkills } from "../../Bladeburner/Skills";
 import { SkillNames } from "../../Bladeburner/data/SkillNames";
+import { Bladeburner } from "../../Bladeburner/Bladeburner";
 
 const bigNumber = 1e27;
 
-export function Bladeburner(): React.ReactElement {
-  if (!Player.bladeburner) return <></>;
-  const bladeburner = Player.bladeburner;
-
+function BladeburnerUI({ bladeburner }: { bladeburner: Bladeburner }): React.ReactElement {
```

Comment:
I don't like this name tbh... maybe aliasing import for `Bladeburner` game feature would be better than renaming this UI component.

Reply:
1. I had to create this wrapper because just moving the "return empty fragment block" to the end of the component would cause NPEs in many places (`Player.bladeburner` can be null)

2. I like the second point though. In some of my previous projects there was even a rule that react component cannot return an empty fragment, null, or whatever else. I.e. if a component won't be able to render stuff - just don't ask it to do so, don't render it as a child of another component. But this approach has a downside in that a parent component needs to know how rendering is done by the child component. In our case, it should be fine though.

Reply:
BTW, it's even already the case - in `DevMenu.tsx`:
```tsx
{Player.bladeburner && <BladeburnerElem />}
```
So I'll just add it as a prop and wrapper won't be needed. But I still need a good name either for import alias or for UI component, since they are the same :)

Reply:
P.S. existing `BladeburnerElem` does not sound nice to me xD

Reply:
Yeah there's a conflict between the BladeburnerRoot (root component of the actual bladeburner UI), Bladeburner (an ingame class), and whatever this is.

I think the most descriptive name is something like BladeburnerDevOptions. While it's long, it clearly explains in the name what its purpose is and it's clear that it's not related to either of the other two Bladeburners above. It doesn't need to have different names in the two files (previously Bladeburner in one file, BladeburnerElem in the other)

Reply:
Re: the useStates not being able to be moved to the top, I don't think any of those rely on Player.bladeburner so I'm not sure I see what the conflict would have been

Reply:
> I think the most descriptive name is something like BladeburnerDevOptions

I like having a name like this. The only thing I don't like is that it's inconsistent with other "dev" components.
Maybe we can rename them all (as well as the files they are located in) to something like `XxxDev` (BladeburnerDev, CompaniesDev, etc)?

Reply:
Yeah that would be fine for consistency